domains/bryanparedes.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": "Bryan-Paredes"
+    },
+    "record": {
+        "A": "76.76.21.21"
```

```suggestion
        "A": ["76.76.21.21"]
```
A records must be in an array.